worldedit-bukkit/adapters/adapter-1_20_5/src/main/java/com/sk89q/worldedit/bukkit/adapter/ext.fawe/v1_20_R4/PaperweightDataConverters.java

```diff
@@ -81,16 +80,16 @@
 @SuppressWarnings({ "rawtypes", "unchecked" })
 public class PaperweightDataConverters extends DataFixerBuilder implements com.sk89q.worldedit.world.DataFixer {
 
-    //FAWE start - BinaryTag
+    //FAWE start - LinTag
```

Comment:
FAWE does not diff from Upstream here (anymore)
https://github.com/EngineHub/WorldEdit/blob/version/7.3.x/worldedit-bukkit/adapters/adapter-1.20.6/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/v1_20_R4/PaperweightDataConverters.java#L88

(At least for `fixUp(FixType, T, int)`, the actual `fix...` methods use fully qualified imports)



Applies to the remaining DataConverters as well